tests/device_tests/test_msg_applysettings.py

```diff
@@ -421,3 +421,16 @@ def test_label_too_long(client: Client):
     with pytest.raises(exceptions.TrezorFailure), client:
         client.set_expected_responses([messages.Failure])
         device.apply_settings(client, label="A" * 33)
+
+
+@pytest.mark.models(skip=["legacy", "safe3"])
+@pytest.mark.setup_client(pin=None)
+def test_set_brightness(client: Client):
+    with client:
+        IF = InputFlowSetBrightness(client)
+        client.set_input_flow(IF.get())
```

Comment:
wouldn't this work even without the input flow?

Reply:
https://github.com/trezor/trezor-firmware/pull/4378/commits/8f513951a85af39d95d06fef29f6fc7de967c0d3